Extensions/xkit_patches.js

```diff
@@ -1859,6 +1859,17 @@ XKit.tools.getParameterByName = function(name){
 				m_return.is_mine = $(obj).hasClass("is_mine");
 				m_return.is_following = ($(obj).attr('data-following-tumblelog') === true);
 				m_return.can_edit = $(obj).find(".post_control.edit").length > 0;
+				
+				if (m_return.is_reblogged)
+				{
```

Comment:
The brace should be on the line above to remain consistent with the rest of the file